samples/apps/logging/logging.cpp

```diff
@@ -460,6 +460,15 @@ namespace loggingapp
         PUBLIC_RECORDS, context, 10000, 20);
       context.get_indexing_strategies().install_strategy(index_per_public_key);
 
+      // According to manual obvervation it's enough to start evicting old
+      // requests on historical perf test, but not to small to get stuck because
+      // of a single request being larget than the cache.
+      //
+      // A much better approach would be to create a new endpoint for
```

Comment:
That does seem like a thing to do in runtime options, can we ticket it?

Reply:
#6299